File: Data Structures & Algorithms/majority-element/submission-0.java
class Solution {
    public int majorityElement(int[] nums) {
     HashMap<Integer, Integer>hm=new HashMap<>();
     for(int n:nums){
        hm.put(n,hm.getOrDefault(n,0)+1);
     }
     int maxval=Collections.max(hm.values());
    for(HashMap.Entry<Integer,Integer>hmentry:hm.entrySet()){
        if(hmentry.getValue().equals(maxval)){
            return hmentry.getKey();
        }
    }
    return -1;
    }
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        count={}
        n=len(nums)
        
        for item in nums:
            count[item]=count.get(item,0)+1
        maxvals=[k for k,v in count.items() if v>(n//3)]
        return maxvals